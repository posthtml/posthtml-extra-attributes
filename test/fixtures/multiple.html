<div>First</div>
<p>Second</p>
