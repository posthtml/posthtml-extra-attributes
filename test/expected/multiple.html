<div class="test" role="section">First</div>
<p class="test" role="section">Second</p>
